src/components/DndTable/Table.js

```diff
@@ -214,11 +185,16 @@ class EnhancedTable extends React.Component {
 
   componentDidMount() {
     const columnData = localStorage.getItem("Cols")
+    // console.log(JSON.parse(localStorage.getItem("Cols")));
     this.setState({
       dataCopy: this.props.data,
       renderer: this.props.data,
-      ...(columnData && {columnData: JSON.parse(columnData)})
+      ...(columnData && {columnData: JSON.parse(localStorage.getItem("Cols"))}),
+     columnData: this.props.name === "Profile Items" ? localStorage.getItem("Cols") ? JSON.parse(localStorage.getItem("Cols")) :this.props.columnData : localStorage.getItem("Tcols") ? JSON.parse(localStorage.getItem("Tcols")) : this.props.columnData
```

Comment:
use variable columnData declared at line 187 instead of using localStorage.getItem("Cols")

Reply:
done